src/System.Net.ServicePoint/tests/ServicePointManagerTest.cs

```diff
@@ -114,6 +114,7 @@ public static void ReusePort_Roundtrips()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Desktop default SecurityProtocol to Ssl3; explicitly changed to SystemDefault for core.")]
```

Comment:
.NET 4.7's default was changed to `SystemDefault` according to the [release notes](https://github.com/Microsoft/dotnet/blob/master/releases/net47/dotnet47-changes.md#networking) (I haven't verified). Will we want this test to run on .NET Framework when the tests run on 4.7?